src/services/rfqm_service.ts

```diff
@@ -228,6 +238,12 @@ export class RfqmService {
             return null;
         }
 
+        // Get the makerUri
```

Comment:
Apologies I can't remember what we agreed upon, are we going to run the RFQ balance checker process here or did we decide to punt?

Reply:
I vaguely recall us putting a starter task on our backlog for something similar - doing an eth call on the order to check balances - but not sure if that's the same thing